12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,44 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+l */[a-z]*
+
 
   2. All files whose name starts with an uppercase letter
   Answer:
+l */[A-Z]*
+
 
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
 
+l */*[aeiou]
+
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
+l */*[!aeiou]
+
 
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
 
   6. All files with extension ".c" under any folder
   Answer:
+echo */*[c]
 
 2. Write a command that would delete all files ending with a ".png" under any subdirectory.
 
+rm */*.*[png]
```

Comment:
Here the square brackets are a real mistake. You'll delete all files ending in a `p`, an `n` or a `g`, while you should only delete those ending in the sequence `.png`